components/resizable-iframe/index.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * Imported from Calypso, with some lint fixes and gutenburg specific changes.
+ */
```

Comment:
:woman_shrugging: I mean, it's a direct copy, but with some react changes for how it handles the references, and lint fixes.